flang/include/flang/Lower/Allocatable.h

```diff
@@ -90,6 +97,12 @@ void associateMutableBoxWithShift(Fortran::lower::FirOpBuilder &,
                                   const fir::ExtendedValue &source,
                                   mlir::ValueRange lbounds);
 
+void associateMutableBoxWithShift(
```

Comment:
Doxygen? Since this is the .h file. (Same for rest. To try to avoid grief when upstreamig.)